### DISCUSSION TITLE
USD Skeleton and Maya Skin cluster Export Issue

### DISCUSSION THREAD

**Discussion by HamedPourkerman** [Autodesk/maya-usd#1746] [Q&A]
Hi,

Issue: 
When I export a simple skeleton with the Skin cluster using Maya 2022 USD exporter, The skin cluster is broke in the USD file.

To narrow down the problem and work on a standard file: 

1) I downloaded the Pixar USDSKEL Example from here https://graphics.pixar.com/usd/downloads.html
2)Imported the USD as Maya nodes (via Import USD) and Animated the "USDSKEL Example" joints and baked it

Scenario 1:
3) Then I exported it to USD and it works perfectly fine

Scenario 2:
4) I removed the skin cluster from the meshes and selected the geometries and the skeleton and applied a maya skin cluster to it
5)After I exported it to USD with the same settings, now the mesh is going everywhere like crazy

Note: the only thing that has changed is that I removed the weird USD skin clusters node and Bind them again to the joints with the Maya skin Cluster.

Any hint or guide toward fixing this would be very much appreciated.

I don't know if this is the right place to report a Maya USD exporter Issue or not, if it's not I am apologizing and please let me know where to put this post?

Best,
Hamed Pourkerman

**Comment by williamkrick** [comment]
Hi @HamedPourkerman,

This is the right place to ask about the MayaUSD exporter. The Import/Export code for USD is a part of the MayaUSD plugin, which version of the plugin are you testing?

I need more details about your workflow to understand why the behavior is not what you expected it to be.

I tried your workflows in Maya 2022 with the current tip of the MayaUSD dev branch. Scenario 1 works correctly for me as well.

In scenario 2 I'm confused about why you removed the skin clusters that was created on import. The skin cluster nodes that are created by import _are_ Maya skin clusters. Can you elaborate a little bit about why you felt this step was necessary? At a high level, there should be very little difference between the skin clusters created when you import and the skin clusters generated by Maya when re-binding the meshes to the skeleton. The skin weights will all be different, and the binding needs to be done carefully to ensure the right joints influence the right meshes. It seems like a lot of work for an unclear gain?

When I delete the skin clusters the rig jumps back to a T-pose, which doesn't match the current skeleton's pose. What was your workflow to rebind all the meshes to the joints?
